sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -306,3 +360,64 @@ def test_limit_val_batches_is_int(required_args_reference, limit_val_batches):
     arglist = parse_kwargs_to_arglist(required_args_reference)
     parser = get_parser()
     parser.parse_args(arglist)
+
+
+
+
+
+@pytest.mark.slow
+def test_pretrain_cli_with_tensorboard(tmpdir, data_path):
```

Comment:
similarly for this test, lets merge it with test_pretrain_cli - please add the tensorboard parameters and test if it is executed